Functions/xASL_init_DetermineRequiredPaths.m

```diff
@@ -34,12 +34,21 @@
     % BIDS DatasetRoot directory
     x.dir.DatasetRoot = x.opts.DatasetRoot;
     x.opts.dataParType = 'directory';
+    
+    % Set other basic BIDS directories
+    x.dir.SourceData = fullfile(x.opts.DatasetRoot,'sourcedata');
```

Comment:
Nice to have all in `x.dir`. I don't understand the difference between `x.opts` and `x.dir` here. Is x.opt necessary as a separate input options field? Don't all input options fall under some other field?

Reply:
Don't think this is the right issue/PR or the right time to restructure this again.